src/toil/common.py

```diff
@@ -1268,6 +1273,36 @@ def getToilWorkDir(configWorkDir: Optional[str] = None) -> str:
             raise RuntimeError(f'The directory specified by --workDir or TOIL_WORKDIR ({workDir}) does not exist.')
         return workDir
 
+    @staticmethod
+    def get_working_tmpdir(tmpdir_prefix: Optional[str] = None) -> Optional[str]:
+        """
+        Get a working temp directory, testing if it is accessible
+        If tmpdir_prefix is given, assuming it is accessible, it will return its parent directory
+        if the path itself is not a directory path
```

Comment:
I might recommend creating a directory with that prefix. Then we could obey it.

Reply:
I agree with that. The only downside is there is no easy way without a refactor to just tack on the `workflow_id` onto the prefix path and use that as the coordination directory. The current solution tacks the workflow_id on but then lets `get_local_workflow_coordination_dir` create another subdirectory with the same workflow_id name. This is since the batchsystem cleanup depends on `get_toil_coordination_dir` to return the parent directory of the coordination files, where all the coordination directory logic is. So in the current setup, I think I have to return a full directory path. I think the only way to respect the prefix probably is to either carry arguments down the function call (as I did) or carry return values up (so `get_local_workflow_coordination_dir` would understand when `get_toil_coordination_dir` returns a prefix/is not a full directory path)